_includes/charts/tigera-operator/templates/tigera-operator/02-role-tigera-operator.yaml

```diff
@@ -23,6 +23,28 @@ rules:
       - update
       - delete
       - watch
+  - apiGroups:
```

Comment:
Should we remove this first block since it's more permissive than the second?

Reply:
I found that this first block was necessary.  It seems the operator will get, list and watch the top level resourcequota.  

At least I get errors if I don't have this block.